File: Array and String/Reverse_string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        char temp;
        for(int i = 0, j = s.size()-1; j > i; i++, j--)
        {
            temp = s[i];
            s[i] = s[j];
            s[j] =  temp;
        }
    }
};

File: Study Plan/Algorithm I/Reverse_String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        i = 0
        while(i < len(s) / 2):
            s[i], s[(len(s) - 1 - i)] = s[(len(s) - 1 - i)], s[i]
            i += 1
        